python/paddle/v2/framework/tests/test_expand_op.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+import numpy as np
+from op_test import OpTest
+
+
+class TestExpandOpRank1(OpTest):
+    def setUp(self):
+        self.op_type = "expand"
+        self.inputs = {'X': np.random.random(12).astype("float32")}
+        self.attrs = {'expandTimes': [2]}
+        output = np.tile(self.inputs['X'], 2)
+        self.outputs = {'Out': output}
+
+    def test_check_output(self):
+        self.check_output()
+
+    def test_check_grad(self):
+        self.check_grad(['X'], 'Out')
+
+
+class TestExpandOpRank2(OpTest):
+    def setUp(self):
+        self.op_type = "expand"
+        self.inputs = {'X': np.random.random((12, 14)).astype("float32")}
+        self.attrs = {'expandTimes': [1, 1]}
+        output = np.tile(self.inputs['X'], (1, 1))
```

Comment:
I think use all one as expand time is not good for test ~

Reply:
Yeah, this is a corner case and I will add more robust test cases. Same as bellow.